src/timeline/TimelineViewManager.h

```diff
@@ -1,69 +1,80 @@
-/*
```

Comment:
I don't know if I'm comfortable removing mujx's copyright notices.  Perhaps it could be modified to include something about contributions by the Nheko Reborn project in addition to his original copyright.

Reply:
Oops, that actually was an accident. I actually started with a completely new file in the timeline2 folder and just implemented the functions that were called from the outside. Only in one of the last few commits I deleted the old timeline folder and renamed the timeline2 folder to timeline. So the final diff looks like I changed the file. Since I didn't actually copy anything, I forgot to add the copyright header, I should add that back...